drgn/helpers/linux/runq.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2025, Oracle and/or its affiliates.
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+"""
+Run Queues
+----------
+The ``drgn.helpers.linux.runq`` module provides helpers for working with the
+Linux run queues.
+"""
+from drgn import Object, Program
+from drgn.helpers.linux.percpu import per_cpu
+from drgn.helpers.linux.sched import task_cpu
+
+
+def runq_clock(prog: Program, cpu: int) -> int:
+    """
+    Get clock of cpu runqueue ``struct rq``
+
+    :param prog: drgn program
+    :param cpu: cpu index
+    :returns: cpu runqueue clock in ns granularity
+    """
+    rq = per_cpu(prog["runqueues"], cpu)
+    return rq.clock.value_()
+
+
+def task_lastrun2now(task: Object) -> int:
+    """
+    Get the duration from task last run timestamp to now
```

Comment:
I had to read the kernel code to figure out whether this timestamp was for when a task last started running or stopped running. I'd be even more explicit here with something like "Get the number of nanoseconds since a task last started running."